staging/src/k8s.io/api/README.md

```diff
@@ -1,3 +1,7 @@
+> ⚠️ **This is a staging repository for Kubernetes**.  
```

Comment:
We should maybe link to what "staging repository" means since it's not a standard term.